ReGoap/Unity/ReGoapAgent.cs

```diff
@@ -15,6 +15,8 @@ public class ReGoapAgent<T, W> : MonoBehaviour, IReGoapAgent<T, W>, IReGoapAgent
 
         public bool CalculateNewGoalOnStart = true;
 
+        public bool shouldDebugPlan = false;
```

Comment:
As Written before, this should probably go in the Planner settings

Reply:
Got it, will make changes according to the reviews;